src/antares/model/binding_constraint.py

```diff
@@ -91,6 +91,19 @@ def generate_id(cls, data: Union[Dict[str, str], LinkData, ClusterData]) -> str:
             return "%".join(sorted((data.area1.lower(), data.area2.lower())))
         return ".".join((data.area.lower(), data.cluster.lower()))
 
+    def serialize_term_data(self) -> Optional[str]:
+        """
+        Serializes the term data to be correctly written in INI.
+        """
+        if isinstance(self.data, LinkData):
+            if self.offset is not None:
+                return f"0.000000%{self.offset}"
+            if self.weight is not None:
+                return f"{self.weight}"
+            return "0"
+
+        return None
```

Comment:
I just realized there's already a method doing this who don't have the issues I told in a previous reivew so you should use it : `weight_offset` inside TermOperators class